arm-sql/2015-05-01/swagger/sql.json

```diff
@@ -1,256 +1,167 @@
+                        "schema": {
+                            "$ref": "#/definitions/ArmErrorResponseMessage"
+                        }
+                    }
+                },
+                "deprecated": false
+            }
         }
-      ],
-      "description": "Represents an Azure SQL Server Security Alert Policy."
     },
-    "Resource": {
-      "properties": {
-        "id": {
-          "readOnly": true,
-          "type": "string",
-          "description": "Resource Id"
+    "definitions": {
+        "ResourceMoveDefinition": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": "string"
+                }
+            }
         },
-        "name": {
-          "readOnly": true,
-          "type": "string",
-          "description": "Resource name"
+        "ArmErrorResponseMessage": {
+            "type": "object",
+            "properties": {
+                "error": {
+                    "$ref": "#/definitions/ArmErrorResponseExtendedErrorInfo"
+                }
+            }
         },
-        "type": {
-          "readOnly": true,
-          "type": "string",
-          "description": "Resource type"
+        "ArmErrorResponseExtendedErrorInfo": {
+            "properties": {
+                "details": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ArmErrorResponseErrorDetail"
+                    }
+                },
+                "innerError": {},
+                "code": {
+                    "type": "string"
+                },
+                "message": {
+                    "type": "string"
+                },
+                "target": {
+                    "type": "string"
+                }
+            }
         },
-        "location": {
-          "type": "string",
-          "description": "Resource location"
+        "ArmErrorResponseErrorDetail": {
+            "type": "object",
+            "properties": {
+                "code": {
+                    "type": "string"
+                },
+                "message": {
+                    "type": "string"
+                },
+                "target": {
+                    "type": "string"
+                }
+            }
         },
-        "tags": {
-          "type": "object",
-          "additionalProperties": {
-            "type": "string"
-          },
-          "description": "Resource tags"
+        "ArmResourceSku": {
+            "type": "object",
+            "properties": {
+                "name": {
```

Comment:
any required properties?
please make those properties an enum where it is possible to do so.